client/src/index.css

```diff
@@ -1,6 +1,13 @@
 @import url('https://fonts.googleapis.com/css2?family=Clicker+Script&family=Esteban&family=Montserrat:wght@400;700&display=swap');
 
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
 body {
+  background-color: darkgrey;
```

Comment:
Generalnie to możemy to usunąć, ogólny zamysł storybooka był taki, że to w nim mamy komponenty i tam widać jak one wygląda, Klara pokazała jak przenieść to na widok, więc w teorii nie powinnniśmy sobie na razie zaśmiecać tego App, ale jak już raz pozwoliłem to już wszyscy to tam dajemy. Dlatego może tego nie dawajmy bo nic to nam nie daje

Reply:
gotowe